app/code/core/Mage/Api/Model/Wsdl/Config.php

```diff
@@ -53,7 +53,7 @@ public static function getNamespacesPrefix()
     }
 
     /**
-     * @return Varien_Simplexml_Config_Cache_Abstract|Zend_Cache_Core
+     * @return Mage_Core_Model_Cache|Varien_Simplexml_Config_Cache_Abstract|null
```

Comment:
I think this should just be `Varien_Simplexml_Config` now that we also changed `Varien_Simplexml_Config`, right?

Reply:
mmm are you sure? the getCache method strictly returns Mage_Core_Model_Cache,  c623fac0b1

am I missing something?

Reply:
Sorry I had made a typo and updated the comment. I also meant that we can only return `Mage_Core_Model_Cache`.

Reply:
Also here:
app/code/core/Mage/Api/Model/Config.php
app/code/core/Mage/Core/Model/Config.php

Reply:
done it in f8de2a9a6e :-)